src/api/models/mermaid.py

```diff
@@ -1412,16 +1412,20 @@ def _set_species_agg_vals(self):
         self._biomass_b = None
         self._biomass_c = None
         self._max_length = None
-        if family.get("biomass_constant_a") is not None:
-            self._biomass_a = round(family.get("biomass_constant_a"), 6)
-        if family.get("biomass_constant_b") is not None:
-            self._biomass_b = round(family.get("biomass_constant_b"), 6)
-        if family.get("biomass_constant_c") is not None:
-            self._biomass_c = round(family.get("biomass_constant_c"), 6)
-        if family.get("max_length") is not None:
-            self._max_length = round(family.get("max_length"), 6)
+        self._regions = []
 
-        self._regions = FishFamily.regions_agg.get(str(self.pk))
+        if family is not None:
+            if family.get("biomass_constant_a") is not None:
```

Comment:
is this a safer check?

`if family.get("biomass_constant_a"):`

It would at least skip over it if the value happens to be zero